rep-0105/ECEF_ENU_Longitude_Latitude_relationships.svg

```diff
@@ -0,0 +1,348 @@
+     id="path5105"
+     style="font-size:16px" /><path
+     d="m 51.49594,541.02374 8.010721,2.16021 -0.329523,1.22197 -1.139001,-0.30714 c 0.721357,0.82566 0.932194,1.79416 0.632512,2.90551 -0.130189,0.48275 -0.336606,0.90314 -0.619253,1.26118 -0.282659,0.35801 -0.59191,0.59963 -0.927754,0.72485 -0.335855,0.1252 -0.708232,0.17582 -1.117135,0.15187 -0.264999,-0.0175 -0.711791,-0.11103 -1.340375,-0.28053 l -4.925613,-1.32826 0.366137,-1.35775 4.872811,1.31402 c 0.553154,0.14916 0.981002,0.20789 1.283548,0.17621 0.302537,-0.0317 0.571798,-0.16004 0.807785,-0.385 0.235977,-0.22497 0.402786,-0.51848 0.500428,-0.88054 0.155941,-0.57831 0.106983,-1.1269 -0.146876,-1.64579 -0.253869,-0.51889 -0.893728,-0.91665 -1.919578,-1.19328 l -4.37497,-1.17978 z"
+     id="path5107"
+     style="font-size:16px" /></g>
+<text
+   x="315.86932"
+   y="-413.14511"
+   transform="scale(1,-1)"
+   id="text3721-8-2-5-1"
+   xml:space="preserve"
+   style="font-size:16px;font-style:normal;font-weight:normal;fill:#000000;fill-opacity:1;stroke:none;font-family:sans-serif"><tspan
+     x="315.86932"
+     y="-413.14511"
+     id="tspan3727-8-4-5-7">φ</tspan></text>
+
+<text
+   x="292.20465"
+   y="-379.30618"
+   transform="scale(1,-1)"
+   id="text3721-8-2-5-1-1"
+   xml:space="preserve"
+   style="font-size:16px;font-style:normal;font-weight:normal;fill:#000000;fill-opacity:1;stroke:none;font-family:sans-serif"><tspan
+     x="292.20465"
+     y="-379.30618"
+     id="tspan3727-8-4-5-7-1">λ</tspan></text>
+
+</g></svg>
```

Comment:
Github will render it if you ask for the file directly: https://github.com/ros-infrastructure/rep/blob/c582b335786a3d51c328c3f52de7cf9c93521c81/rep-0105/ECEF_ENU_Longitude_Latitude_relationships.svg